src/main/java/org/jitsi/jicofo/auth/ShibbolethHandler.java

```diff
@@ -53,6 +53,8 @@
 
     private final ShibbolethAuthAuthority shibbolethAuthAuthority;
 
+    private static final String SHIBBOLETH_TARGET = "/login";
```

Comment:
I feel bad for asking about it, but can you move it above the `shibbolethAuthAuthority` so that `static final` are grouped together?

Reply:
sure, added a javadoc as well.